src/client/client_http.go

```diff
@@ -108,14 +108,19 @@ func DownloadNetwork(httpClient *http.Client, hostname string, networkPath strin
 	if err != nil {
 		return err
 	}
-	defer r.Body.Close()
 
-	out, err := os.Create(networkPath)
+	out, err := ioutil.TempFile("", "lczero_tmp")
```

Comment:
Seems the recommendation is to create tempfiles in the same directory you are going to move them to rather than in the default directory, since os.Rename doesn't work across devices.